org.lflang/src/org/lflang/generator/ts/TSGenerator.kt

```diff
@@ -223,21 +214,12 @@ class TSGenerator(
      * as the source file, copy a default version from $LIB_PATH/.
      */
     private fun copyConfigFiles() {
-        for (configFile in CONFIG_FILES) {
-            val configFileDest = fileConfig.srcGenPath.resolve(configFile)
-            val configFileInSrc = fileConfig.srcPath.resolve(configFile)
-            if (configFileInSrc.toFile().exists()) {
-                println("Copying $configFileInSrc to $configFileDest")
-                Files.createDirectories(configFileDest.parent)
-                Files.copy(configFileInSrc, configFileDest, StandardCopyOption.REPLACE_EXISTING)
```

Comment:
Why did this feature get removed? It used to copy user-supplied build files.

Reply:
Oh, you're right. That happened by accident. I didn't realize that it was also doing that. Thanks for the catch